src/portalbackend/PortalBackend.DBAccess/Models/CompanyApplicationWithStatus.cs

```diff
@@ -25,6 +25,8 @@ namespace Org.Eclipse.TractusX.Portal.Backend.PortalBackend.DBAccess.Models
     public class CompanyApplicationWithStatus
     {
         public Guid ApplicationId;
-        public CompanyApplicationStatusId? ApplicationStatus; //FIXME - this should not be nullable!
+        public CompanyApplicationStatusId ApplicationStatus;
+        public IEnumerable<ApplicationChecklistData>? ApplicationChecklistDatas;
```

Comment:
ApplicationChecklistData should not be nullable

```suggestion
        public IEnumerable<ApplicationChecklistData> ApplicationChecklistData;
```